app/controllers/v0/evss_claims_controller.rb

```diff
@@ -3,6 +3,7 @@
 module V0
   class EVSSClaimsController < ApplicationController
     include IgnoreNotFound
+    service_tag 'claim-status'
```

Comment:
This one isn’t associated with the claims status stuff. Not sure who uses it anymore honestly

Reply:
Even if it were not used, the goal is to tag all controllers that are rendered as active routes in vets-api, so they can be inventoried. And this definitely logically goes with the claim-status service.

Also, it is being used - looks like the `#request_decision` action on this controller is receiving traffic in prod - [Loki Query](https://grafana.vfs.va.gov/explore?orgId=1&left=%5B%221679198400000%22,%221679284799000%22,%22Loki%20(Prod)%22,%7B%22expr%22:%22%7Bapp%3D%5C%22vets-api-server%5C%22%7D%20%7C%3D%20%5C%22EVSSClaimsController%5C%22%5Cn%5Cn%22,%22hide%22:false%7D%5D)

Reply:
Oh yeah, that’s true, forgot that method exists